src/dbally/_main.py

```diff
@@ -36,7 +38,7 @@ def create_collection(
         name: Name of the collection is available for [Event handlers](event_handlers/index.md) and is\
         used to distinguish different db-ally runs.
         llm: LLM used by the collection to generate responses for natural language queries.
-        event_handlers: Event handlers used by the collection during query executions. Can be used to\
+        override_event_handlers: Event handlers used by the collection during query executions. Can be used to\
```

Comment:
Let's call it `event_handlers` and mention here that it will override global handlers defined elsewehere.